app/views/overview/_rc.html

```diff
@@ -1,9 +1,9 @@
-<div class="deployment-tile" ng-if="deployment.kind === 'ReplicationController'">
+<div class="deployment-tile">
```

Comment:
just curious, will this rc partial work for petset too now that its agnostic to kind?